### PR TITLE
Adding tests for the docker containers (deployment still has to be added)

### PR DESCRIPTION
**Pull request by KaiSzuttor**

### CODE REVIEW

**Comment by KaiSzuttor** [comment]
@fweik please activate the tests for the docker repo on travis.

**Comment by fweik** [comment]
Fixes #2.

**Comment by fweik** [comment]
Fixes #12.

File: docker/ubuntu-cuda/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN apt-get update && apt-get install -y \
     libhdf5-openmpi-dev \
     doxygen \
     vim \
-&& pip2 install cython sphinx sphinxcontrib-bibtex numpydoc --upgrade \
+&& pip install --upgrade pip; pip2 install cython sphinx sphinxcontrib-bibtex numpydoc --upgrade \
```
Comment (fweik) [inline docker/ubuntu-cuda/Dockerfile]: Why do you upgrade pip (3), and then use pip2?

Comment (KaiSzuttor) [reply docker/ubuntu-cuda/Dockerfile]: The default is pip2 here

**Comment by fweik** [comment]
Then use pip in both cases? Also a commit will trigger travis.

**Comment by fweik** [comment]
Lgtm

**Comment by fweik** [comment]
Hm, this takes to long...

**Comment by KaiSzuttor** [comment]
I hope this is fixed with the job splitting

**Comment by fweik** [comment]
Looks good.

**Comment by KaiSzuttor** [comment]
The failing job should be fixed by removing the cuda library dependency (see PR https://github.com/espressomd/espresso/pull/1698)

**Pull request merged**